server/routes/wishCards.js

```diff
@@ -252,6 +328,36 @@ router.get('/me', renderPermissions, async (req, res) => {
   }
 });
 
+// @desc    Render wishcards edit page
+// @route   GET '/wishcards/edit/:id'
+// @access  Private, verified partner
+// @tested 	No
+router.get('/edit/:id', limiter, renderPermissions, async (req, res) => {
+  try {
+    const wishcard = await WishCardRepository.getWishCardByObjectId(req.params.id);
+    const agency = wishcard.belongsTo;
+    const { agencyAddress } = agency;
+
+    const childBirthday = moment(wishcard.childBirthday).format('YYYY-MM-DD');
+
+    if (res.locals.user.userRole !== 'admin') {
+      const userAgency = await AgencyRepository.getAgencyByUserId(res.locals.user._id);
+      if (String(wishcard.belongsTo._id) !== String(userAgency._id)) {
+        return res.status(404).render('404');
+      }
+    }
+    res.render('wishcardEdit', { wishcard, agencyAddress, childBirthday }, (error, html) => {
+      if (error) {
+        res.status(400).json({ success: false, error });
```

Comment:
this renders a blank page with just json on it

render a different page here with some info for the user instead of plain json

make it pretty